model/communication/AppRoute.ttl

```diff
@@ -36,8 +38,25 @@ ids:AppRoute a owl:Class;
     	idsm:forProperty ids:appRouteBroker;
     	idsm:relationType idsm:OneToMany ;
     ];
+    idsm:validation [
+         idsm:forProperty ids:appRouteClearingHouse;
+         idsm:relationType idsm:OneToMany ;
+    ];
+
 .
 
+ids:RouteStep a owl:Class;
+              rdfs:subClassOf ids:AppRoute;
+              rdfs:label "Route step"@en ;
+              rdfs:comment """
```

Comment:
Why is this a multi-line comment with triple """ ? The line break after "hold" looks confusing.

Reply:
Multiline comments in turtle are written using either """ or '''
See https://www.w3.org/TR/turtle/#turtle-literals

But you are right. The line breaks after """ look a bit confusing. Removed them in 52d2897